app/assets/stylesheets/styleguide/_typography.scss

```diff
@@ -36,6 +36,7 @@
 // CORE FONTS - NEW TRANSPORT
 
 @mixin core-48($line-height: (50 / 48), $line-height-640: (30 / 25)) {
+  // scss-lint:disable NameFormat
```

Comment:
I'd expect this file to be redundant given what's in frontend toolkit. Looks like there's a piece of work to update this app to _the modern way_.